rundeckapp/grails-spa/packages/ui-trellis/src/app/components/job/resources/NodeTable.vue

```diff
@@ -90,34 +94,42 @@
                       style="margin: 0 5px"
                       filter-key="name"
                       :filter-val="node.nodename!"
+                      data-test-id="node-name-filter"
                       @nodefilterclick="filterClick"
                     >
                       <i class="glyphicon glyphicon-circle-arrow-right"></i>
                     </node-filter-link>
 
                     <span class="nodedesc"></span>
+
                     <span class="text-strong">
                       <i
                         v-for="badge in glyphiconBadges(node.attributes)"
                         v-if="node.attributes['ui:badges']"
```

Comment:
Adding here the reply I've sent on slack:

> there’s nothing wrong with using v-if and v-for in the same element since Vue 3 was released. Docs don’t recommend it because it’s not obvious what takes precedence just by reading the code, but the answer is that the v-if takes precedence. So the code first checks if the if passes, if it does, then executes the for loop.